src/helpers/matchers/matchTextContent.ts

```diff
@@ -1,15 +1,25 @@
 import { Text } from 'react-native';
 import type { ReactTestInstance } from 'react-test-renderer';
+import { getConfig } from '../../config';
 import { matches, TextMatch, TextMatchOptions } from '../../matches';
 import { filterNodeByType } from '../filterNodeByType';
 import { getTextContent } from '../getTextContent';
 
-export function matchTextContent(
-  node: ReactTestInstance,
-  text: TextMatch,
-  options: TextMatchOptions = {}
-) {
-  if (!filterNodeByType(node, Text)) {
+type MatchTextContentParams = {
+  node: ReactTestInstance;
+  text: TextMatch;
+  options?: TextMatchOptions;
+};
+
+export function matchTextContent({
```

Comment:
All other matchers user positional arguments. What arguments are they for going with objects as args in this case?

Reply:
I refactored it when it had typeToMatch arg as well so it was a bit too much arguments for positional arguments. I personally am not a huge fan of positional arguments, I find the calls to be more explicit with objects and also when there are multiple optional params it becomes quite hard to handle with positional arguments

Reply:
Bringing up this point again. If we want to check the positional arguments to JS names arguments, lets do it for all queries for consistency, and in a separate PR.